### PR TITLE
Add joke to bot

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

Add joke to bot

## Issue reference

Add joke to bot

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by amanbha** [comment]
@youngbupark joke-parsing issue :) https://github.com/dapr/dapr/actions/runs/111981866

File: .github/workflows/dapr-bot.yml [hunk 1]
```diff
@@ -38,6 +38,25 @@ jobs:
             const isFromPulls = !!payload.issue.pull_request;
             const commentBody = payload.comment.body;
 
+            if (commentBody && commentBody.indexOf("/make-me-laugh") == 0) {
+              const result = await github.request("https://official-joke-api.appspot.com/random_joke");
+
+              if (!result || !result.data) {
+                return;
```
Comment (shalabhms) [inline .github/workflows/dapr-bot.yml:45]: have a default joke or some fun text :-) instead of just return :-)

**Comment by shalabhms** [comment]
I just added review comment, have a default joke :-) even in error, so bot will still make you laugh ...

**Comment by LMWF** [comment]
"i have a bad feeling about this"

**Pull request merged**